lib/rock.rb

```diff
@@ -0,0 +1,14 @@
+class Rock
+  attr_reader :value
+  def initialize
+    @value = 'rock'
+  end
+
+  def win_status(opponent_choice)
+    if [@value, Paper.new.value, Spock.new.value].include?(opponent_choice)
```

Comment:
Should the shape class be deciding if it wins? I'm not sure if another class is better or if the classes should know if they win or lose. I like the logic that you've used to compare them.